src/plugins/intel_gna/tests/functional/shared_tests_instances/subgraph_tests/convolution_relu_sequence.cpp

```diff
@@ -17,14 +17,15 @@ namespace {
 class GnaConvolutionReluSequenceTest : public ConvolutionReluSequenceTest, GnaLayerTestCheck {
 protected:
     void Run() override {
-        GnaLayerTestCheck::SkipTestCheck();
-
-        if (!GnaLayerTestCheck::skipTest) {
-            ConvolutionReluSequenceTest::Run();
+        if (gnaLibVersionLessThan(2.1f)) {
```

Comment:
IMO we dont expect the GNA LIB  will be less than 2.1

Reply:
ok